### PR TITLE
Added information and CSS to Intro page and Home Page

### PR DESCRIPTION
**Pull request by Cast02**

### CODE REVIEW

File: multimedia.py [hunk 1]
```diff
@@ -35,15 +47,15 @@
 
 
 
-@app.route('/')
+
```
Comment (KrispyKris905) [inline multimedia.py:50]: wait don't we need @app.route('/')  ?

File: multimedia.py [hunk 2]
```diff
@@ -35,15 +47,15 @@
...
 def home():
     return render_template('index.html')
 
 @app.route('/new_User')
 def newPage():
     return render_template('introPage.html')
 
-@app.route('/returing_User', methods=['GET', 'POST'])
+@app.route('/', methods=['GET', 'POST'])
```
Comment (KrispyKris905) [inline multimedia.py:58]: should this stay as it is for returning users. Also something minor is the spelling.

File: multimedia.py (hunk 1 above)
Comment (Cast02) [reply multimedia.py:50]: if you look on line 58 of the multimedia.py. The login page become the new @app.route("?")

File: multimedia.py (hunk 2 above)
Comment (Cast02) [reply multimedia.py:58]: no that becomes the new home route

**Review by FabianSantano — APPROVED** [review]
Looks good

**Pull request merged**